test/cfg/wxwidgets.cpp

```diff
@@ -36,6 +37,32 @@
 #include <wx/position.h>
 #include <wx/versioninfo.h>
 
+void unreadVariable_wxPoint2DInt(const wxInt32 x, const wxPoint2DInt& pti, const wxPoint &pt)
+{
+    // cppcheck-suppress unreadVariable
```

Comment:
This should be `unusedVariable` (also below)
See https://github.com/danmar/cppcheck/pull/6154

Reply:
@chrchr-github Thanks!
Now it passes the checks with on my side so far

```
$ ../../cppcheck --check-library --library=wxwidgets --enable=style,information --inconclusive --error-exitcode=1 --disable=missingInclude --inline-suppr wxwidgets.cpp
Checking wxwidgets.cpp ...
Checking wxwidgets.cpp: __VISUALC__...
Checking wxwidgets.cpp: __WXGTK__;__WXMSW__...
Checking wxwidgets.cpp: __WXOSX__...
Checking wxwidgets.cpp: wxUSE_DEBUG_CONTEXT=1...
Checking wxwidgets.cpp: wxUSE_GUI=1...
Checking wxwidgets.cpp: wxUSE_GUI=1;wxUSE_SPINCTRL=1...
Checking wxwidgets.cpp: wxUSE_SPINCTRL=1...
nofile:0:0: information: Active checkers: 171/592 (use --checkers-report=<filename> to see details) [checkersReport]
```
I am wondering why the github-workflow did not fail before?

Reply:
Unmatched suppressions are currently not reported, so the CI will pass with either `unusedVariable` or `unreadVariable`, since no warning is emitted at all. `unusedVariable` will be correct though once #6157 is merged.

Reply:
Ok, thanks for clarifying.